scripts/prebuild/mdx-transforms/build-mdx-transforms.mjs

```diff
@@ -24,9 +24,12 @@ import {
 	loadRedirects,
 } from './rewrite-internal-redirects/rewrite-internal-redirects.mjs'
 import { transformExcludeTerraformContent } from './exclude-terraform-content/index.mjs'
+import { transformExcludeVaultContent } from './exclude-vault-content/index.mjs'
 
 import { PRODUCT_CONFIG } from '#productConfig.mjs'
 
+export const DIRECTIVE_PRODUCTS = ['VLT', 'TFC', 'TFEnterprise']
```

Comment:
@williamdalessandro let's move this into the exclude-terraform-content transform. That logic is specific to there. I think you moved it here so it's "easier to find" but we should keep all of the logic for the transformed self contained.

Reply:
Looking over this PR a bit more I think I see why you did this. You separated out the TF/TFE/HCP TF exclusions from the Vault exclusions. In that case I think it should be in it's own file; something like so:

```
scripts/prebuild
- ...
- exclude-content
  - constants.mjs
  - exclude-terraform-content.mjs
  - exclude-vault-content.mjs
  - ...
```

Reply:
@williamdalessandro one more thought here. Let's use `Vault` instead just `VLT`. Mainly for the reason that from my understanding we don't never use VLT to mean Vault. Whereas TFC and TFE are both commonly used. It's just a few extra characters that I think will make things a bit clearer for the education team.